central/reports/scheduler/schedule.go

```diff
@@ -119,8 +119,13 @@ type ReportRequest struct {
 }
 
 type reportEmailFormat struct {
-	WhichVulns string
-	DateStr    string
+	ImageFlavorBranding string
```

Comment:
No, this is not branding, and this is not related to image flavor. It is just branded product name.

```suggestion
	BrandedProductName string
```

Feel free to give it another name.

Reply:
Done